index.js

```diff
@@ -168,6 +168,10 @@ module.exports = {
     // Disabling this rule until this is resolved https://github.com/yannickcr/eslint-plugin-react/issues/1848
     // at the moment the fix makes the code look messy and at times unreadable
     'react/jsx-one-expression-per-line': 'off',
+
+    // The only thing these AirBnB-enabled rules do is create unnecessary boilerplate
+    'react/require-default-props': 'off',
+    'react/default-props-match-prop-types': 'off',
```

Comment:
I get the first one, and agree. However, I'd like to see the reasoning for this one. It doesn't seem to be addressed like the first in your description

Reply:
You're right. The second rule is fine, it's the first one that's the problem. I'll remove the second one.